src/Settings.hs

```diff
@@ -66,3 +66,23 @@ findPackageByName name = find (\pkg -> pkgName pkg == name) knownPackages
 -- | Install's DESTDIR setting.
 getDestDir :: Action FilePath
 getDestDir = fromMaybe "" <$> cmdInstallDestDir
+
+-- | Arguments to run GHC's test script.
+getTestArgs :: Args
```

Comment:
Why is this defined here and not in `Settings.Builders.RunTest` together with the rest of the flags?

Reply:
I define this method in `src/Setting.hs` to translate the arguments from command line to `Args` type.

Without this function I'll need to import module `CommandLine` into `Settings.Builders.RunTest` to do the transformation. I don't sure if it's the right way (import `CommandLine` into the builder module).

Reply:
The `getTestArgs` returns the arguments provided by user from command line. In `Settings.Builders.RunTest` we only process the arguments decided by current GHC's configuration.

Reply:
We use `Settings.Builders.X` to define all X-specific arguments. This seems to be relevant only for `RunTest`.

Reply:
> I don't sure if it's the right way (import CommandLine into the builder module).

Yes, that's fine, as long as this doesn't create nasty import cycles. In fact, we do something very similar in `Settings.Builders.Make` where we rely on `shakeThreads <$> expr getShakeOptions` which essentially comes from Hadrian's command line (but captured by Shake).

Reply:
I have moved the `getTestArgs` from `Settings` into `Settings.Builders.RunTest` and deleted the unused `cmdTestArgs` function.